src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx

```diff
@@ -68,21 +46,21 @@ export const ItemFlow = ({ name, index, onRemove }: ItemFlowProps) => {
   return (
     <ToggleItemContainer
       title={title}
-      Content={Content}
-      HeaderContent={Actions}
+      Content={ItemFlowContent}
+      HeaderContent={ItemFlowActions}
       contentProps={{
-        items: conditions,
         name: itemName,
+        conditions,
         onRemove: handleRemoveCondition,
-        conditionalError,
+        'data-testid': dataTestid,
```

Comment:
'data-testid': dataTestid,
        'data-testid': `${dataTestid}-content`,

Reply:
it is used just for the actions like Add/Remove/Collapse so there is no sense to separate header and content